Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -230,35 +230,46 @@ extension MemberBlockItemListSyntax {
 
 extension IfConfigDeclSyntax {
     func model(with encloserAcl: String, declKind: NominalTypeDeclKind, metadata: AnnotationMetadata?, processed: Bool) -> (Model, String?, Bool) {
-        var subModels = [Model]()
+        var clauses: [IfMacroModel.Clause] = []
         var attrDesc: String?
         var hasInit = false
+        for (index, cl) in self.clauses.enumerated() {
+            let clauseType: IfMacroModel.Clause.ClauseType
+            if cl.condition == nil {
+                clauseType = .else
+            } else if index == 0 {
+                clauseType = .if
+            } else {
+                clauseType = .elseif(order: index)
+            }
```

Comment:
It think using `tokenKind` is more better way

```suggestion
            let clauseType: IfMacroModel.Clause.ClauseType
            switch cl.poundKeyword.tokenKind {
            case .poundIf:
                clauseType = .if
            case .poundElse:
                clauseType = .else
            case .poundElseif:
                clauseType = .elseif(order: index)
            default:
                fatalError("unexpected tokenKind: \(cl.poundKeyword.tokenKind)")
            }
```